fdbserver/Knobs.h

```diff
@@ -497,6 +497,14 @@ class ServerKnobs : public Knobs {
 	int64_t FASTRESTORE_ATOMICOP_WEIGHT; // workload amplication factor for atomic op
 	int64_t FASTRESTORE_APPLYING_PARALLELISM; // number of outstanding txns writing to dest. DB
 	int64_t FASTRESTORE_MONITOR_LEADER_DELAY;
+	int64_t FASTRESTORE_STRAGGLER_THRESHOLD;
+	bool FASTRESTORE_TRACK_REQUEST_LATENCY;
```

Comment:
I meant `FASTRESTORE_STRAGGLER_THRESHOLD` but probably misplaced the line. Ideally, it's better to add a suffix "_SECONDS" to be more explicit. The same applies to `FASTRESTORE_WAIT_FOR_MEMORY_LATENCY` and `FASTRESTORE_HEARTBEAT_MAX_DELAY`.

Reply:
For the delay and latency knobs, I think the conventional way used in the code base is to omit the seconds, say `STORAGE_LOGGING_DELAY`. That is because by default the `delay` is in seconds. I can add SECONDS to `FASTRESTORE_STRAGGLER_THRESHOLD ` knob.